perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -48,19 +49,25 @@
                     'invocation of this test and then clean-up the bucket '
                     'after this test returns.')
 
+flags.DEFINE_string('azure_account', None,
+                    'The name of the storage account for Azure.')
+
+flags.DEFINE_string('azure_key', None,
+                    'The key of the storage account for Azure.')
+
 flags.DEFINE_enum(
     'scenario', 'OneByteRW', ['OneByteRW', 'ListConsistency',
-                              'SingleStreamThroughput'],
+                              'SingleStreamThroughput', 'CleanupBucket'],
     'The various scenarios to test. OneByteRW: read and write of single byte. '
     'ListConsistency: List-after-write and list-after-update consistency. '
-    'SingleStreamThroughput: Throughput of single stream large object RW. (not '
-    'implemented yet)')
+    'SingleStreamThroughput: Throughput of single stream large object RW. '
+    'CleanupBucket: Cleans up everything in a given bucket.')
```

Comment:
CleanupBucket doesn't seem to be a scenario to test. Consider change the comments to redefine the scenario flag.